appointment/reschedule/clientReschedule.js

```diff
@@ -0,0 +1,74 @@
+require.config({
```

Comment:
I wouldn't mind comments on the name of these files. You think `clientReschedule` is a good term? Or should we do something like `selfServiceReschedule`?

Reply:
@CollinsSpencer I decided to go with `selfServiceAppointmentReschedule` instead of `clientReschedule`--I think this new name describes what it is much better.